sdk/data/wrapper/wrapper.go

```diff
@@ -153,6 +156,9 @@ func (w *Wrapper) updateDataPartition(isInit bool) (err error) {
 	localLeaderPartitionGroups := make([]*DataPartition, 0)
 	for _, partition := range dpv.DataPartitions {
 		dp := convert(partition)
+		if w.followerRead && w.nearRead {
```

Comment:
It cannot be sorted so simply. When updatingDataParttion, because dp.Host must be in the same order in the master-slave replication agreement. Unchangeable

Reply:
add NearHosts to save sorted hosts